rust_crate/src/interface.rs

```diff
@@ -7,13 +7,18 @@ use super::interface_os::*;
 #[cfg(target_family = "wasm")]
 use super::interface_web::*;
 
+#[cfg(feature = "bevy")]
+use bevy_ecs::event::Event;
+
 /// This is a mutable cell type that can be shared across threads.
 pub type SharedCell<T> = OnceLock<Mutex<RefCell<Option<T>>>>;
 
 /// This contains a message from Dart.
 /// Optionally, a custom binary called `binary` can also be included.
 /// This type is generic, and the message
 /// can be of any type declared in Protobuf.
+/// If the bevy feature is used, every message can be received as an event in bevy.
```

Comment:
A small request here, do you mind change the sentene to:

If the `bevy` feature is used, every message can be received as an event in Bevy.

Reply:
I just did it. I hope everything fits.